admin/partials/perfecty-push-admin-dashboard.php

```diff
@@ -78,51 +78,59 @@
 
 </script>
 <div class="wrap">
-	<h1>Perfecty Push</h1>
-	<h2>Plugin stats</h2>
-	<div class="notice notice-notice"><p>Welcome to Perfecty Push. Start off by subscribing from the <a href="<?php echo site_url(); ?>">front page</a>
-			and send your first <a href="<?php echo admin_url( 'admin.php?page=perfecty-push-send-notification' ); ?>">notification</a>!</p></div>
+	<h1><?php printf( esc_html__( 'Perfecty Push', 'perfecty-push-notifications' ) ); ?></h1>
+	<h2><?php printf( esc_html__( 'Plugin stats', 'perfecty-push-notifications' ) ); ?></h2>
+	<div class="notice notice-notice"><p>
+	<?php
+	/* translators: %1$s: Link to site front page; %2$s: Link to notifications admin page */
+	printf(
+		__( 'Welcome to Perfecty Push. Start off by subscribing from the %1$s and send your first %2$s', 'perfecty-push-notifications' ),
+		'<a href="' . site_url() . '">' . esc_html__( 'front page', 'perfecty-push-notifications' ) . '</a>',
+		'<a href="' . admin_url( 'admin.php?page=perfecty-push-send-notification' ) . '">' . esc_html__( 'notification', 'perfecty-push-notifications' )
+	);
+	?>
+	</a>!</p></div>
 	<div style="width: 800px;">
 		<canvas id="daily-notifications"></canvas>
 	</div>
 	<div class="perfecty-push-stats">
 		<div>
-			<h3>Users</h3>
+			<h3><?php printf( esc_html__( 'Users', 'perfecty-push-notifications' ) ); ?></h3>
 			<div class="perfecty-push-stats-text">
-				<span>Total users: </span><span><?php echo $users_stats['total']; ?></span>
+				<span><?php printf( esc_html__( 'Total users:', 'perfecty-push-notifications' ), $users_stats['total'] ); ?> </span><span><?php echo $users_stats['total']; ?></span>
 			</div>
 			<div class="perfecty-push-stats-text">
-				<span>Active: </span><span><?php echo $users_stats['active']; ?></span>
+				<span><?php printf( esc_html__( 'Active:', 'perfecty-push-notifications' ) ); ?> </span><span><?php echo $users_stats['active']; ?></span>
 			</div>
 			<div class="perfecty-push-stats-text">
-				<span>Inactive: </span><span><?php echo $users_stats['inactive']; ?></span>
+				<span><?php printf( esc_html__( 'Inactive:', 'perfecty-push-notifications' ) ); ?> </span><span><?php echo $users_stats['inactive']; ?></span>
 			</div>
 		</div>
 		<div>
-			<h3>Notifications</h3>
+			<h3><?php printf( esc_html__( 'Notifications', 'perfecty-push-notifications' ) ); ?></h3>
 			<div class="perfecty-push-stats-text">
-				<span>Total notifications: </span><span><?php echo $notifications_stats['total']; ?></span>
+				<span><?php printf( esc_html__( 'Total notifications:', 'perfecty-push-notifications' ) ); ?>< /span><span><?php echo $notifications_stats['total']; ?></span>
```

Comment:
This was showing `Total notifications: < /span>178`, but I'll fix it directly :)